src/ralph/ssl_certificates/tests/tests.py

```diff
@@ -0,0 +1,116 @@
+import os
+from io import StringIO
+from unittest.mock import MagicMock, patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from ralph.assets.models import Manufacturer
+from ralph.ssl_certificates.models import CertificateType, SSLCertificate
+
+
+class ImportSSLCertificatesTest(TestCase):
+    def setUp(self):
+        self.base_dir = os.path.dirname(
+            os.path.dirname(__file__)
+        )
+
+    def test_command_output(self):
+        out = StringIO()
+        call_command('import_ssl_certificates', '404', stderr=out)
+        self.assertIn('', out.getvalue())
+
+    def test_wildcard_certificate_domain_ssl_should_by_without_prefix(self):  # noqa
+        out = StringIO()
+        dir = os.path.join(
```

Comment:
1. ``dir`` is built-in function, please rename to ``samples_dir``
2. move ``samples_dir`` to ``setUp`` method

Reply:
Done